### PR TITLE
Fix: APP-1148 - Default Empty Resource Row

### PR DESCRIPTION
**Pull request by Fabricevladimir**
## Description
- adds empty row to resource list when creating a new proposal to the following flows: main proposal, manage members, mint tokens, settings, and withdraw.

Task: [APP-1148](https://aragonassociation.atlassian.net/browse/APP-1148)

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)

## Checklist:

- [x] I have selected the correct base branch.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.
- [x] Any dependent changes have been merged and published in downstream modules.
- [ ] I ran all tests with success and extended them if necessary.
- [ ] I have updated the ``CHANGELOG.md`` file in the root folder.
- [x] I have tested my code on the test network.

### CODE REVIEW

**Review by sepehr2github — APPROVED** [review]
Thanks Fabrice, LGTM 🚀

File: packages/web-app/src/app.tsx [hunk 1]
```diff
@@ -151,6 +151,7 @@ function App() {
 const NewSettingsWrapper: React.FC = () => {
   const formMethods = useForm({
     mode: 'onChange',
+    defaultValues: {links: [{name: '', url: ''}]},
```
Comment (WissenIstNacht) [inline packages/web-app/src/app.tsx:154]: Are you sure this one is necessary, given that it is overwritten with DAO data?

**Review by WissenIstNacht — COMMENTED** [review]
Looks good, thanks.

**Review by WissenIstNacht — APPROVED** [review]
Pardon, I misclicked. Should have been and approval.

File: packages/web-app/src/app.tsx (hunk 1 above)
Comment (Fabricevladimir) [reply packages/web-app/src/app.tsx:154]: Yeah. It's for the proposal being created form and not the one filled out by the DAO data

**Pull request merged**